packages/react-native/React/Views/RCTCursor.m

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// [macOS]
+
+#import <Foundation/Foundation.h>
+#import "RCTCursor.h"
+
+#if defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && __MAC_OS_X_VERSION_MAX_ALLOWED >= 150000 /* __MAC_15_0 */
+#define __MAC_OS_15_SDK_AVAILABLE
```

Comment:
We should avoid `__` prefix as they're reserved by compilers:
```suggestion
#define RCT_MAC_OS_15_SDK_AVAILABLE
```

Reply:
Looking through the rest of the macros, It seems that style was unique to `JSCRuntime.cpp`. All the other instances of single underscore prefix defines are gone (redundant and deleted after a min OS bump), so I'll change this.

Reply:
It's a C or C++ standards thing, but since Objective-C is based on C, it's safe to assume that they follow the same rules.